### ISSUE TITLE
Using github fork of project

### ISSUE THREAD

**Issue by gonzalomelov** [layerhq/gradle-git-repo-plugin#17]
I have forked a project from github that is already on Maven Central.

After adding the required configuration to use this plugin, I have to add the compile statement on the build gradle file. However the compile statetment is the same as the original project one, so the original ones is used instead of the fork I want to add.

Is there a flag or config parameter to the compile statement to decide from which repository the dependency should be fetched?

Thanks

**Comment by drapp** [comment]
no, gradle doesn't work that way. For each dependency you add to `compile` or `testCompile`, it searches through your list of repositories for matching pom/jars. That includes any github repos added via this plugin. If you've got this set up right you shouldn't need to do anything special in your `dependencies` block
